mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.h

```diff
@@ -89,23 +117,27 @@ std::unique_ptr<Pass> createPreSparsificationRewritePass();
 // The Sparsification pass.
 //===----------------------------------------------------------------------===//
 
+using sparse_tensor::LoopOrderingStrategy;
```

Comment:
there are not many instances following, so I have a slight preference just using the namespace below

LoopOrderingStrategy is something that could easily refer to some other module, so having the namespace helps with that mental note